docs/dev/new_check_howto.md

```diff
@@ -458,6 +458,8 @@ To install your integration, build your Python wheel first (see the [Building](#
 datadog-agent integration install -w /path/to/wheel.whl
 ```
 
+Note that it may require to use a specific user to run that command, such as `dd-agent`.
```

Comment:
I'd put the note before the command cause a lot of people execute commands and then continue reading :p